lib/rucio/daemons/conveyor/submitter.py

```diff
@@ -104,7 +104,7 @@ def submitter(once=False, rses=None, partition_wait_time=10,
         timeout = None
 
     try:
-        bring_online = config_get('conveyor', 'bring_online')
+        bring_online = config_get_int('conveyor', 'bring_online')
```

Comment:
I think the same change needs to be made in the stager as well.

Reply:
Good catch !